modules/Hoot/tools/folderTree.js

```diff
@@ -534,6 +539,12 @@ export default class FolderTree extends EventEmitter {
         this.update( d );
     }
 
+    checkMultiSelectFolder( d ) {
```

Comment:
should this be called `ellipsizeFolderNames`?
what should it return if not more than one selected?  or it's not selected?  or it's type is not folder?

Reply:
changed method name to better reflect purpose